tensorflow/core/kernels/mkl/mkl_matmul_op_fused.cc

```diff
@@ -314,6 +317,7 @@ class MklFusedMatMulOp : public MklDnnMatMulOpBase<T, T> {
   std::vector<string> fused_ops_;
   const int kInputIndex_Add = 3;
   const int kOutputIndex_Dst = 0;
+  const int kWeightTensorHashLength = 1024;
```

Comment:
Nit: Please wrap this in `#ifdef DNNL_AARCH64_USE_ACL`.

Reply:
ditto